roles/openshift_facts/library/openshift_facts.py

```diff
@@ -56,6 +56,13 @@ def migrate_docker_facts(facts):
     if 'node' in facts and 'portal_net' in facts['node']:
         facts['docker']['hosted_registry_insecure'] = True
         facts['docker']['hosted_registry_network'] = facts['node'].pop('portal_net')
+
+    # log_options was originally meant to be a comma separated string, but
+    # we now prefer an actual list, with backward compatability:
+    if 'log_options' in facts['docker'] and \
+            isinstance(facts['docker']['log_options'], basestring):
+        facts['docker']['log_options'] = facts['docker']['log_options'].split(",")
```

Comment:
Ok I think I see it now, init_local_facts is calling migrate for the pre-existing local facts, but this is coming in via "facts_to_set" in this case and not a part of that migration. I have attempted to update, let me know how that looks.